client/cz-clock-dash.html

```diff
@@ -0,0 +1,108 @@
+<link rel="import" href="../bower_components/polymer/polymer.html">
+<link rel="import" href="../bower_components/paper-card/paper-card.html">
+<link rel="import" href="../bower_components/paper-item/paper-item.html">
+
+<dom-module id="cz-clock-dash">
+
+  <template>
+    <style>
+      paper-card .card-flex {
+        display: flex;
+        align-content: center;
+      }
+      paper-item {
+        box-sizing: border-box;
+        width: 80%;
+      }
+      div {
+        width: 300px;
+      }
+    </style>
+
+    <paper-card id='card' heading='Time around the world'>
+      <div class="card-content">
+        <div class='card-flex'>
+          <template is="dom-repeat" items="{{timezones}}">
+            <paper-item-body two-line>
+              <div id="timer" inner-h-t-m-l=[[item.data]]></div>
+              <div id="city" secondary>{{item.city}}</div>
+            </paper-item-body>
+          </template>
+        </div>
+      </div>
+    </paper-card>
+  </template>
+
+  <script>
+    Polymer({
+      is: "cz-clock-dash",
+
+      properties: {
+        timezones: {
+          type: 'Object',
+          notify: true,
+          value: function() { return []; },
+          observer: "timezonesChanged"
+        }
+      },
+      attached: function() {
+        registerSource('cz-config', 'timezones', function(timezones) {
+          this.set('timezones', timezones);
+        }.bind(this));
+      },
+      currentTimeDay: function(timezone) {
+        var today = new Date();
+        var day = today.getUTCDay();
+        // FIXME: The UTC is accurate as of 15 March - what about daylightsavings causing these to change?
+        var hours = today.getUTCHours() + parseFloat(timezone.utc);
+        var minutes = today.getUTCMinutes();
+        var seconds = today.getUTCSeconds();
+
+        if (hours < 0) {
+          hours = 24 + hours;
+          day = day > 0 ? (day - 1) %  7 : 6;
+        } else if (hours >= 24) {
+          hours = hours - 24;
+          day = (day + 1) %  7;
+        }
+
+        var meridian = "am";
+        if (hours >= 12) {
+          hours = hours > 12 ? hours - 12 : hours;
+          meridian = "pm";
+        }
+        hours = (hours < 10) ? "0" + hours : hours;
+        minutes = (minutes < 10) ? "0" + minutes : minutes;
+        seconds = (seconds < 10) ? "0" + seconds : seconds;
+
+        if (day != 0 && day != 6)
+          return this.formattedTime(hours, minutes, seconds, meridian);
+        else
+          return "<span style='color: red'>" + this.intToDay(day) + "</span>";
+      },
+      mutateTimeDay: function(timezone, idx) {
+        this.set('timezones.' + idx + '.data', this.currentTimeDay(timezone));
+        this.async(()=>this.mutateTimeDay(timezone, idx), 500);
```

Comment:
> why is this async?

Oh, this is the loop. 500ms seems a bit fast? Don't we only need to update once a minute? It also appears that when the timezones in the source change we will end up with multiple loops running.